src/components/Select/AutosuggestMulti/SuggestionTag/SuggestionTag.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { MdClose } from 'react-icons/md';
+import { bem } from '../../../../utils';
+import styles from './SuggestionTag.scss';
+
+const { block, elem } = bem('SuggestionTag', styles);
+
+type Props = {
+    /** Tag Stretches on full width */
+    isStretched?: boolean;
+    /** Tag has restricted width */
+    isBounded?: boolean;
+    /** Clicked on delete button */
+    onClick?: (e: React.MouseEvent) => void;
+    /** Tag content */
+    children: NotEmptyReactNode;
+};
+
+export const SuggestionTag: React.FC<Props> = (props) => {
+    const { children, isBounded, isStretched, onClick, ...rest } = props;
+
+    return (
+        <div {...rest} {...block({ isBounded, isStretched })}>
+            <span title={children} {...elem('label')}>
+                {children}
```

Comment:
with the current interface `children` could be a `div` or anything else, which would result in incorrect HTML. Maybe make change `span` to `div`

Reply:
yes, it makes sense to use `div`